test/_assets/samples/ckeditor-dev-correct/samples/old/api.html

```diff
@@ -202,8 +202,7 @@ <h1 class="samples">
 			CKEditor - The text editor for the Internet - <a class="samples" href="http://ckeditor.com/">http://ckeditor.com</a>
 		</p>
 		<p id="copy">
-			Copyright &copy; 2003-2012, <a class="samples" href="http://cksource.com/">CKSource</a> - Frederico
-			Knabben. All rights reserved.
+			Copyright &copy; 2003-2022, <a class="samples" href="http://cksource.com/">CKSource</a> Holding sp. z o.o. All rights reserved.
```

Comment:
Use HTTPS (fix in all samples):

```suggestion
			Copyright &copy; 2003-2022, <a class="samples" href="https://cksource.com/">CKSource</a> Holding sp. z o.o. All rights reserved.
```